tools/find_device.h

```diff
@@ -0,0 +1,14 @@
+#ifndef BL_TOOLS_FIND_DEVICE_H
+#define Bl_TOOLS_FIND_DEVICE_H
+
+/* The path name of the matched /dev node */
+char dev_node[300];
```

Comment:
I don't think this will work.  Each file that includes this header will have its own copy of dev_node.

Reply:
+1
should have thought of extern, but I am confused why there is no link time error reported, and the program worked!